examples/vae/train.py

```diff
@@ -157,15 +157,17 @@ def main(argv):
   rng = random.PRNGKey(0)
   rng, key = random.split(rng)
 
-  train_ds = tfds.load('binarized_mnist', split=tfds.Split.TRAIN)
+  ds_builder = tfds.builder('binarized_mnist')
+  ds_builder.download_and_prepare()
+  train_ds = ds_builder.as_dataset(split=tfds.Split.TRAIN)
```

Comment:
Any reason why you use tfds.Split.TRAIN here and 'train' in your other PR?
https://github.com/google/flax/pull/200/files

Reply:
No, these are same things.